src/EPR.SubsidiaryBulkUpload.Application/Services/CompaniesHouseDownload/FileDownloadService.cs

```diff
@@ -31,7 +31,7 @@ public class FileDownloadService(HttpClient httpClient, ILogger<FileDownloadServ
             }
             catch (HttpRequestException ex)
             {
-                logger.LogError(ex, "Failed to find or connect to the download service (file may not exist) {FilePath}", path);
+                logger.LogError(ex, "Failed to find or  connect to the download service (file may not exist) {FilePath}", path);
```

Comment:
Remove one space from "or  connect" - I think this came from the last merge from main